common/djangoapps/enrollment/api.py

```diff
@@ -250,7 +251,30 @@ def get_course_enrollment_details(course_id):
         }
 
     """
-    return _data_api().get_course_enrollment_info(course_id)
+    cache_key = "enrollment.course.details.{course_id}".format(course_id=course_id)
+
+    try:
+        cached_enrollment_data = cache.get(cache_key)
+    except Exception:
+        # The cache backend could raise an exception (for example, memcache keys that contain spaces)
+        log.exception("Error occurred while retrieving course enrollment details from the cache")
+        cached_enrollment_data = None
```

Comment:
Nitpick: I don't think this line is necessary.  If an exception is raised, `cache.get()` won't return.